packages/salesforcedx-vscode-core/package.json

```diff
@@ -198,7 +198,7 @@
       "editor/context": [
         {
           "command": "sfdx.force.function.containerless.start",
-          "when": "sfdx:project_opened && resource =~ /.*/functions/.*/[^/]+(/[^/]+\\.(ts|js|java|json|toml))?$/ && resourceFilename != package.json && resourceFilename != package-lock.json && resourceFilename != tsconfig.json"
+          "when": "sfdx:project_opened && resource =~ /.*\\/functions\\/.*\\/[^\\/]+(\\/[^\\/]+\\.(ts||js||java||json||toml))?$/ && resourceFilename != package.json && resourceFilename != package-lock.json && resourceFilename != tsconfig.json"
```

Comment:
So here, you're right that `/` needs to be escaped within the regular expression and become `\\/`, but `|` remains same. I guess this error message was misleading: 

<img width="769" alt="image" src="https://user-images.githubusercontent.com/16353531/228358211-79e9d247-e83d-4f5d-b09c-213159624f45.png">

but it says what it says because of unescaped slashes. 

I think this should be the correct when clause in this case (just slashes escaped): 

```
sfdx:project_opened && resource =~ /.*\\/functions\\/.*\\/[^\\/]+(\\/[^\\/]+\\.(ts|js|java|json|toml))?$/ && resourceFilename != package.json && resourceFilename != package-lock.json && resourceFilename != tsconfig.json
```

Reply:
Right! Thank you for taking the time to review this PR🙌🏼